get-involved.php

```diff
@@ -29,9 +29,9 @@
         <div class="whole" id="sections-menu">
             <a class="button flat suggested-action" href="#funding">Funding</a>
             <a class="button flat suggested-action" href="#translations">Translations</a>
-            <a class="button flat suggested-action" href="#support">Support</a>
-            <a class="button flat suggested-action" href="#web-development">Web</a>
-            <a class="button flat suggested-action" href="#desktop-development">Desktop</a>
+            <a class="button flat suggested-action" href="#user-support">User support</a>
```

Comment:
I'm not really sure I agree here because the nav link is also labeled "Support". If you go to Apple, Microsoft, Dell, Google, etc it's always just called "Support".